#! /bin/sh
patch -p1 -l -f $* < $0
exit $?

From d0965eab6ed140da8f0a29b5d55ddd3f50d56ab3 Mon Sep 17 00:00:00 2001
From: Andre Heinecke <aheinecke@gnupg.org>
Date: Thu, 23 Mar 2023 09:02:34 +0100
Subject: [PATCH] Remove unused include

ktoolinvocation_win does not use QDBusConnection. This fixes
the build without DBus for Windows.
---
 src/kdeinit/ktoolinvocation_win.cpp | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/kdeinit/ktoolinvocation_win.cpp b/src/kdeinit/ktoolinvocation_win.cpp
index ad3552c7..5dfa3ebb 100644
--- a/src/kdeinit/ktoolinvocation_win.cpp
+++ b/src/kdeinit/ktoolinvocation_win.cpp
@@ -14,7 +14,6 @@
 #include <KMessage>

 #include <QCoreApplication>
-#include <QDBusConnection>
 #include <QHash>
 #include <QProcess>
 #include <QStandardPaths>
--
2.39.2
